HdmiCECInstance.py

```diff
@@ -0,0 +1,2 @@
+
+instance = None
```

Comment:
Why not just put this in HdmiCec? We have a similar pattern for the infobars

Reply:
I am open to whatever works - I just used this method as its the one used in Navigation(which I know code wise, just never looked at Infobars))  and the method works for python2 & python 3

Reply:
I will kill the Pull and try this in Py3 (no reason it shouldn't work)